### PR TITLE
Removes `microtime` dependency

### PR DESCRIPTION
**Pull request by jaredperreault-okta**
Resolves https://github.com/pubkey/broadcast-channel/pull/1022

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,9 +62,9 @@
     "lint": "eslint src test config --cache",
     "clear": "rimraf -rf ./dist && rimraf -rf ./gen",
     "build:es6node": "rimraf -rf dist/esnode && cross-env NODE_ENV=es6 babel src --out-dir dist/esnode && echo '{ \"type\": \"module\" }' > dist/esnode/package.json",
-    "build:es6browser": "rimraf -rf dist/esbrowser && cross-env NODE_ENV=es6 babel src --out-dir dist/esbrowser && grep -rl NodeMethod dist/esbrowser/ | xargs sed -i 's/.*NodeMethod.*//' && echo '{ \"type\": \"module\" }' > dist/esbrowser/package.json",
+    "build:es6browser": "rimraf -rf dist/esbrowser && cross-env NODE_ENV=es6 babel src --out-dir dist/esbrowser && grep -rl NodeMethod dist/esbrowser | xargs sed -i.bak '/.*NodeMethod.*/d' && echo '{ \"type\": \"module\" }' > dist/esbrowser/package.json",
```
Comment (jaredperreault-okta) [inline package.json:65]: previous `sed` commands will not run on BSD-based distros (like Mac OS X). This should run on BSD and GNU, however I was only able to test on BSD

**Comment by pubkey** [comment]
Can you resolve the conflicts so that i can trigger the ci?
Also please do not commit any generated files from the dist folder.

**Comment by jaredperreault-okta** [comment]
@pubkey I updated my PR, but I cannot trigger the CI workflows myself

**Comment by pubkey** [comment]
Thank you, merged.

**Comment by pubkey** [comment]
I just released `4.16.0` with your changes.

**Pull request merged**